String, RegExp, DataTime.js

```diff
@@ -1,33 +1,32 @@
-//...............1...................
-let strk = "ahb acb aeb aeeb adcb axeb";
 
-let result = strk.match(/ahb/);
-let result2 = strk.match(/acb/);
-let result3 = strk.match(/aeb/);
 
-alert( result);
-alert( result2);
+/////
 
-alert( result3);
 
+//...............1...................
+var str = 'ahb acb aeb aeeb adcb axeb'.replace(/a\w\b\s/);;
+console.log(str)

```

Comment:
No, you need to use Regexp. This is the task for Regexp. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp